src/net/larse/lcms/algorithms/VCT.java

```diff
@@ -71,34 +71,32 @@
   private static final int COMP = 7;
   private static final int N_BANDS = 8;
 
-  private static final List<Integer> UD_INDEXES = Arrays.asList(B3, B5, B7);
-
   // Mask constants - all "fillable" categories are <= 5
   // Even though not all categories are used, we list them for metadata
   // purposes.  Note that these ARE NOT the same categories as in VCT
   // source code.
 
-  //TODO: (yang) consider to change these to enum.
-//  public enum Mask{
-//    BACKGROUND(0),
-//    CLOUD(1),
-//    CLOUD_EDGE(2),
-//    SHADOW(3),
-//    SHADOW_EDGE(4),
-//    SNOW(5),
-//    WATER(6),
-//    CLEAR_LAND(7),
-//    CORE_FOREST(8),
-//    CORE_NONFOREST(9),
-//    CONFIDENT_CLEAR(10),
-//    FILL_CLASSES(5);
-//
-//    private final int value;
-//
-//    Mask(int value) {
-//      this.value = value;
-//    }
-//  }
+  // TODO: (yang) consider to change these to enum.
+  // public enum Mask{
+  //   BACKGROUND(0),
+  //   CLOUD(1),
+  //   CLOUD_EDGE(2),
+  //   SHADOW(3),
+  //   SHADOW_EDGE(4),
+  //   SNOW(5),
+  //   WATER(6),
+  //   CLEAR_LAND(7),
+  //   CORE_FOREST(8),
+  //   CORE_NONFOREST(9),
+  //   CONFIDENT_CLEAR(10),
+  //   FILL_CLASSES(5);
+  //
+  //   private final int value;
+  //
+  //   Mask(int value) {
+  //     this.value = value;
+  //   }
+  // }
```

Comment:
@c11, if you want to implement this as enum, that seems fine to me, although I'm unclear of the benefit.